src/main/java/at/porscheinformatik/seleniumcomponents/clarity/ClarityTextareaContainerComponent.java

```diff
@@ -0,0 +1,39 @@
+package at.porscheinformatik.seleniumcomponents.clarity;
+
+import static at.porscheinformatik.seleniumcomponents.WebElementSelector.*;
+
+import at.porscheinformatik.seleniumcomponents.SeleniumComponent;
+import at.porscheinformatik.seleniumcomponents.WebElementSelector;
+import at.porscheinformatik.seleniumcomponents.component.TextAreaComponent;
+
+public class ClarityTextareaContainerComponent extends ClarityFormControlContainer
+{
+    private final TextAreaComponent textarea = new TextAreaComponent(this);
+
+    // ---
+
+    public static ClarityTextareaContainerComponent within(SeleniumComponent parent)
+    {
+        return new ClarityTextareaContainerComponent(parent, selectByTagName("clr-textarea-container"));
+    }
+
+    public static ClarityTextareaContainerComponent bySeleniumKey(SeleniumComponent parent, String seleniumKey)
+    {
+        return new ClarityTextareaContainerComponent(parent,
+            selectBySeleniumKey("clr-textarea-container", seleniumKey));
+    }
+
+    // ---
+
+    public ClarityTextareaContainerComponent(SeleniumComponent parent, WebElementSelector selector)
+    {
+        super(parent, selector);
+    }
+
+    // ---
+
+    public void enterText(String text)
```

Comment:
I think it makes not much sense to have a TextArea container that can only enter text but can not access it.

Reply:
I'm making it public.